graphql/project_resolver.go

```diff
@@ -25,18 +25,18 @@ func (r *projectResolver) IsFavorite(ctx context.Context, obj *restModel.APIProj
 
 // Patches is the resolver for the patches field.
 func (r *projectResolver) Patches(ctx context.Context, obj *restModel.APIProjectRef, patchesInput PatchesInput) (*Patches, error) {
-	opts := patch.ByPatchNameStatusesCommitQueuePaginatedOptions{
-		Project:         obj.Id,
-		PatchName:       patchesInput.PatchName,
-		Statuses:        patchesInput.Statuses,
-		Page:            patchesInput.Page,
-		Limit:           patchesInput.Limit,
-		OnlyCommitQueue: patchesInput.OnlyCommitQueue,
-		IncludeHidden:   patchesInput.IncludeHidden,
-		Requesters:      patchesInput.Requesters,
+	opts := patch.ByPatchNameStatusesMergeQueuePaginatedOptions{
+		Project:        obj.Id,
+		PatchName:      patchesInput.PatchName,
+		Statuses:       patchesInput.Statuses,
+		Page:           patchesInput.Page,
+		Limit:          patchesInput.Limit,
+		OnlyMergeQueue: patchesInput.OnlyCommitQueue,
```

Comment:
could we also add an optional GraphQL input param `OnlyMergeQueue` and migrate the frontend to use that parameter?